src/css/style.css

```diff
@@ -104,3 +109,80 @@ img {
 .card:hover {
     transform: scale(1.1)
 }
+
+
+
+/* code for service_cards page */
+
+.services_heading {
+display: flex;
+justify-content: center;
+font-weight: bolder;
+font-size: xx-large;
+margin-top: 40px;
+margin-bottom: 40px;
+}
+
+.servicesBox {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;}
+
+
+.servicesPicture {
+    width: 100%;
+    /* border-radius: 12px; */
+    height: 200px;
+    object-fit: cover;
+}
+
+.service_card_container {
```

Comment:
have the card container have display flex; and flex-direction:column so the content spans the remaining height of the card.